### PR TITLE
Forgot password

### PR DESCRIPTION
**Pull request by nvolodymyr**
add UI

### CODE REVIEW

File: src/app/component/auth/sign-in-new/sign-in-new.component.ts [hunk 1]
```diff
@@ -6,7 +6,8 @@ import { HttpErrorResponse } from '@angular/common/http';
 import { UserOwnSignInService } from '../../../service/auth/user-own-sign-in.service';
 import { Router } from '@angular/router';
 import { SignInIcons } from 'src/assets/img/icon/sign-in/sign-in-icons';
-import { MatDialogRef } from '@angular/material';
+import {MatDialog, MatDialogRef} from '@angular/material';
+import {RestorePasswordComponent} from '../restore-password/restore-password.component';
```
Comment (Gusvarov) [inline src/app/component/auth/sign-in-new/sign-in-new.component.ts:10]: add spaces between brackets

File: src/app/component/auth/restore-password/restore-password.component.scss [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+  .wrapper {
+    width: 100%;
+
+    .right-side {
+      width: auto;
+    }
+  }
+}
+
+
```
Comment (KimAT25) [inline src/app/component/auth/restore-password/restore-password.component.scss:174]: delete unneeded enter line

File: src/app/component/auth/restore-password/restore-password.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
```
Comment (KimAT25) [inline src/app/component/auth/restore-password/restore-password.component.spec.ts:2]: delete unneeded enter line

File: src/app/component/auth/restore-password/restore-password.component.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    this.dialog.closeAll();
+  }
+
+  private onBackToSignIn(): void {
+    this.dialog.open(SignInNewComponent, {
+      hasBackdrop: true,
+      closeOnNavigation: true,
+      panelClass: 'custom-dialog-container',
+    });
+
```
Comment (KimAT25) [inline src/app/component/auth/restore-password/restore-password.component.ts:34]: delete unneeded enter line

File: src/app/component/auth/sign-in-new/sign-in-new.component.html [hunk 5]
```diff
@@ -33,7 +33,7 @@ <h2>{{'user.auth.sign-in.fill-form' | translate}}</h2>
         </span>
       </div>
       <div class="forgot-wrapper">
-        <a class="forgot-password">
+        <a (click)="onOpenForgotWindow()"  class="forgot-password">
```
Comment (KimAT25) [inline src/app/component/auth/sign-in-new/sign-in-new.component.html:36]: put attributes in 1 column

File: src/app/component/auth/sign-in-new/sign-in-new.component.ts (hunk 1 above)
Comment (nvolodymyr) [reply src/app/component/auth/sign-in-new/sign-in-new.component.ts:10]: fixed

File: src/app/component/auth/restore-password/restore-password.component.scss (hunk 2 above)
Comment (nvolodymyr) [reply src/app/component/auth/restore-password/restore-password.component.scss:174]: fixed

File: src/app/component/auth/restore-password/restore-password.component.spec.ts (hunk 3 above)
Comment (nvolodymyr) [reply src/app/component/auth/restore-password/restore-password.component.spec.ts:2]: fixed

File: src/app/component/auth/restore-password/restore-password.component.ts (hunk 4 above)
Comment (nvolodymyr) [reply src/app/component/auth/restore-password/restore-password.component.ts:34]: fix

File: src/app/component/auth/sign-in-new/sign-in-new.component.html (hunk 5 above)
Comment (nvolodymyr) [reply src/app/component/auth/sign-in-new/sign-in-new.component.html:36]: fixed

File: src/app/component/auth/restore-password/restore-password.component.scss [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+      border-radius: 4px 0 0 4px;
+    }
+  }
+
+  .right-side {
+    width: 50%;
+    max-width: 480px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
```
Comment (nabermi2020) [inline src/app/component/auth/restore-password/restore-password.component.scss:32]: Please add empty line

File: src/app/component/auth/auth.module.ts [hunk 7]
```diff
@@ -1,19 +1,22 @@
 import { NgModule } from '@angular/core';
```
Comment (nabermi2020) [inline src/app/component/auth/auth.module.ts:1]: general: Please name branches according to the agreed terms of the name(features - feature/*, bugs - bug/*)

File: src/app/component/auth/restore-password/restore-password.component.scss (hunk 6 above)
Comment (nvolodymyr) [reply src/app/component/auth/restore-password/restore-password.component.scss:32]: fixed

File: src/app/component/auth/auth.module.ts (hunk 7 above)
Comment (nvolodymyr) [reply src/app/component/auth/auth.module.ts:1]: ok

**Pull request merged**